ckanext/stadtzhharvest/tests/test_harvester.py

```diff
@@ -351,536 +303,552 @@ def _run_full_job(self, harvest_source_id, num_jobs=1, num_objects=1):
+        )
+        assert test_csv["description"] == u""
 
-        wms = next(r for r in result['resources'] if r["name"] == "Web Map Service") 
-        eq_(wms['description'], u'')
+        wms = next(
+            r for r in result["resources"] if r["name"] == "Web Map Service"
+        )
+        assert wms["description"] == u""
 
-        wfs = next(r for r in result['resources'] if r["name"] == "Web Feature Service") 
-        eq_(wfs['description'], u'Dies ist eine Spezial-Beschreibung')
+        wfs = next(
+            r
+            for r in result["resources"]
+            if r["name"] == "Web Feature Service"
+        )
+        assert wfs["description"] == u"Dies ist eine Spezial-Beschreibung"
 
     def test_harvest_create_with_dataset_prefix(self):
-        data_path = os.path.join(
-            __location__,
-            'fixtures',
-            'test_dropzone'
-        )
-        test_config = json.dumps({
-            'data_path': data_path,
-            'metafile_dir': '',
-            'update_datasets': True,
-            'update_date_last_modified': False,
-            'dataset_prefix': 'testprefix-',
-        })
-
-        results = self._test_harvest_create(1, config=test_config)['results']
-        eq_(len(results), 1)
-        eq_(results[0]['name'], u'testprefix-test_dataset')
-        eq_(results[0]['title'], u'Administrative Einteilungen Stadt Zürich')
-        eq_(results[0]['license_id'], u'cc-by')
-        eq_(results[0]['updateInterval'][0], u'woechentlich')
-        eq_(results[0]['dataType'][0], u'Einzeldaten')
-        eq_(len(results[0]['resources']), 1)
-
-    def test_harvest_create_dwh(self):
-        data_path = os.path.join(
-            __location__,
-            'fixtures',
-            'DWH'
-        )
-        test_config = json.dumps({
-            'data_path': data_path,
-            'metafile_dir': '',
-            'update_datasets': True,
-            'update_date_last_modified': False
-        })
-
-        results = self._test_harvest_create(3, config=test_config)
-        eq_(len(results['results']), 3)
-        for result in results['results']:
-            expected_titles = [
-                u'Geburten nach Jahr, Geschlecht und Stadtquartier',
-                u'Test Nachnamen in der Stadt Zürich',
-                u'Daten der permanenten Velozählstellen - Stundenwerte',
-            ]
-
-            assert result['title'] in expected_titles, "Title does not match result: %s" % result
+        data_path = os.path.join(__location__, "fixtures", "test_dropzone")
+        test_config = json.dumps(
+            {
+                "data_path": data_path,
+                "metafile_dir": "",
+                "update_datasets": True,
+                "update_date_last_modified": False,
+                "dataset_prefix": "testprefix-",
+            }
+        )
+
+        results = self._test_harvest_create(1, config=test_config)["results"]
```

Comment:
Just a question: what happened to the other two datasets that we had as fixtures as it were 3 datasets before?

Reply:
Hm, I think for some reason the diff is showing the tests a bit out of order, probably because there were so many formatting changes. (`flake8` complained so much I just ran `black` over the whole file, and it converts a lot of little things like `'`->`"`.)

This test really hasn't changed - this green section belongs to the new `test_harvest_create_with_dataset_prefix`, and the line with three datasets belongs to the old `test_harvest_create_dwh`. :)

Reply:
I just double-checked in intellij, and it looks correct there.